tests/timezones/tests.py

```diff
@@ -265,6 +236,20 @@ def test_raw_sql(self):
             [event],
             transform=lambda d: d)
 
+    def test_cursor_execute_accepts_naive_datetime(self):
```

Comment:
I'm probably missing something, but it seems like this test and the next one are shadowed by tests of the same name later in the same `TestCase`.